server/services/licenceService.ts

```diff
@@ -120,7 +119,7 @@ export class LicenceService {
 
   async updateLicenceConditions(
     bookingId,
-    existingLicence: Case,
+    existingLicence: LicenceRecord,
```

Comment:
this was slightly wrong type - this only requires licence record rather than a case